### PR TITLE
[1.11.1] 'meteor create --help' command shows --vue and --apollo

### PR DESCRIPTION
**Pull request by sgmheyhey**
'meteor create --help' command is not showing about --vue and --apollo. it could be confused by getting meteor skeleton info.
add help message for meteor create [--vue|--aplollo]

### CODE REVIEW

**Comment by filipenevola** [comment]
Hey, thanks, please sign the CLA @sgmheyhey

File: tools/cli/help.txt [hunk 1]
```diff
@@ -159,7 +159,7 @@ Make a subdirectory named <path> if it doesn't exist and create a new Meteor app
 there. You can pass an absolute path, relative path, or '.' for the current
 directory. Use the --bare option to create an empty app, or the --minimal option
 to create an empty app with as few Meteor packages as possible. To scaffold a
-more complete app, use the --full option. To create a basic React-based app,
+more complete app, use the --full option. To create a basic react-based app,
```
Comment (StorytellerCZ) [inline tools/cli/help.txt:162]: The `R` in React should be capitalized as it is a name of a package/technology.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

**Comment by filipenevola** [comment]
I'm closing this issue because we haven't heard anything from the original poster for a while and we can't merge PRs without a signed CLA.

**Comment by sgmheyhey** [comment]
Signed the CLA. It's late.

**Pull request merged**